usrsctplib/netinet/sctp_pcb.h

```diff
@@ -345,6 +352,7 @@ struct sctp_base_info {
 	void (*debug_printf)(const char *format, ...);
 	int crc32c_offloaded;
 #endif
+        int userland_thread_stats;
```

Comment:
I was thinking, if I need to add something to here for tracing thread status, it would be better to be a single variable than multiple variables(per thread). However, I am not sure if this approach is right.